Client/Frontend/Browser/BrowserViewController/BrowserViewController+URLBarDelegate.swift

```diff
@@ -79,6 +79,22 @@ extension BrowserViewController: URLBarDelegate {
         tabManager.selectedTab?.reload()
     }
 
+    func urlBarDidPressShare(_ urlBar: URLBarView, shareView: UIView) {
+        TelemetryWrapper.recordEvent(category: .action,
+                                     method: .tap,
+                                     object: .awesomebarLocation,
+                                     value: .awesomebarShareTap,
+                                     extras: nil)
+        let arrowDirection: UIPopoverArrowDirection = isBottomSearchBar ? .down : .up
```

Comment:
nit: we could move the arrowDirection inside the if let

Reply:
<img width="970" alt="image" src="https://user-images.githubusercontent.com/8919439/204659603-fa80f15f-13d5-433c-95f2-d675c9a8b78a.png">

Will update it like this